### PR TITLE
Grafana Explore for Prometheus: Collaboration with Daniel Tk

### PR DESCRIPTION
**Pull request by YajB**
# What changed
-updating Grafana Explore topicfor Prometheus: 
https://deploy-preview-924--logz-docs.netlify.app/user-guide/infrastructure-monitoring/grafana-explore-prometheus
- Added a link to this new topic in related Prometheus topics:
   - getting started
   - Final step of Remote write config, in both User Guide and Send Your Data sections. 
- SINGLE SOURCING complex content: Created include snippet for Prometheus Remote Write topic for user guide and SYD content: 
   - Update this in PR #950, to add instructions for HELM charts to the user guide topic.  
      - https://deploy-preview-924--logz-docs.netlify.app/shipping/prometheus-sources/prometheus-remote-write-shipping.html
      - https://deploy-preview-924--logz-docs.netlify.app/user-guide/infrastructure-monitoring/prometheus-remote-write.html

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md [hunk 1]
```diff
@@ -72,8 +85,8 @@ You can click the <i class="fas fa-signal"></i> icon for a top-N analysis of the
 
 
 ### Split and Sync
-
-You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.
+<!-- still relevant-->
+You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.  <!-- what sort of things would you display in the 2 panels, for example?  -->
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md:89]: True - this one's still relevant.
It's especially effective for checking how queries behave in different time ranges and even different data sources (in Logz.io world, data source  = a different metrics account), so we can show something like that

File: _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md [hunk 2]
```diff
@@ -13,21 +13,28 @@ contributors:
   - yberlinger
 ---
 
-Grafana Explore is where you can explore the data available in your Metrics account.
+Grafana Explore is where you can explore the data available in your Prometheus metrics account.
 It's a bit like Kibana Discover, in that it is optimized for quickly searching the data in preparation for creating dashboards.
 
 Whether you just started sending metrics for the first time, or you want to check that your metrics arrived as expected, Grafana’s Explore mode is the best way to do it.
 It's also great if you're a long-time user and want to examine the structure of your metrics documents in order to create a new monitoring dashboard.
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md:20]:
```suggestion
It's also great if you're a long-time user and want to examine the structure of your metrics in order to create a new monitoring dashboard.
```

Comment (daniel-tk) [reply _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md:20]: Now we know P8S metrics are not arranged in docs anymore, as opposed to Elastic.

File: _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md [hunk 3]
```diff
@@ -13,21 +13,28 @@ contributors:
...
 
-To go to Grafana’s Explore, click on the **Explore icon <i class="far fa-compass"></i>** in the left menu.
+To go to Grafana’s Explore, click the **Explore icon <i class="far fa-compass"></i>** in the left menu.
 
-## Metrics view vs. Logs view
+<!-- @DanielT ~~ the next sections are obsolete, I assume.
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md:24]: Exactly

File: _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md [hunk 4]
```diff
@@ -72,8 +85,8 @@ You can click the <i class="fas fa-signal"></i> icon for a top-N analysis of the
 
 
 ### Split and Sync
-
-You can split your Explore mode screen to work faster and make quick comparisons. You can split the screen to work with two views side-by-side, whether Metrics-Metrics, Logs-Logs, or Logs-Metrics.
+<!-- still relevant-->
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/grafana-explore-p8s.md:88]: Right - Still relevant.
It's especially effective for comparing queries from different time ranges or event different data sources (in Logz.io data source = metrics account).

**Review by YajB — COMMENTED** [review]
added animated gifs and clarified that data sources for comparison include both logs or metrics accounts or subaccouts

**Pull request merged**